testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt

```diff
@@ -59,7 +61,11 @@ class MiniAppDisplayActivity : BaseActivity() {
                 }
 
             launch {
-                viewModel.obtainMiniAppView(appId)
+                viewModel.obtainMiniAppView(appId, object: MiniAppMessageBridge() {
+                    override fun getUniqueId(callbackId: String) {
+                        postValue(callbackId, "example_unique_id")
```

Comment:
I think that the call to `postValue` can be move internal to the SDK (probably in `postMessage`?). This will allow the `getUniqueId` function to directly return a string.

Reply:
@corycaywood Previously, I did exactly like that 👍 . So the `postError` should be internal too and only get called from the sdk, right?